### PR TITLE
Update SLES-52573_73973370.pnach

### PR DESCRIPTION
**Pull request by fox140cv**
Add patch unlock fps, tested ok

### CODE REVIEW

**Comment by CookiePLMonster** [comment]
Could you **please** put them in one PR instead of spamming dozens of submissions?

**Comment by fox140cv** [comment]
I'm sorry, can you tell me how to do it please?

**Comment by refractionpcsx2** [comment]
You should learn to use Github Desktop at least, you can do it all through there.

File: patches/SLES-52573_73973370.pnach [hunk 1]
```diff
@@ -25,3 +25,7 @@ patch=1,EE,003f02c4,word,00000000 //34215ffa
 patch=1,EE,00407fdc,word,3c013f2b //3c013f00
 
 
+[50/60 FPS]
```
Comment (refractionpcsx2) [inline patches/SLES-52573_73973370.pnach:28]: 50 or 60?

Comment (fox140cv) [reply patches/SLES-52573_73973370.pnach:28]: In both, both when selecting 50hz and when selecting 60hz

**Comment by refractionpcsx2** [comment]
Also can you make sure your files have the .pnach extension, I'm going to have to go back and fix a whole bunch of them now, because I didn't realise.

**Pull request merged**